source/reference/known-issues.txt

```diff
@@ -68,3 +68,23 @@ Given the exact same |sql| query, the |bi-short| does not return
       GROUP BY class;
       
       Empty set (0.00 sec)
+
+
+Mapping Potentially Null Values at Different Object Depths
+----------------------------------------------------------
+A known mapping error can occur that causes all returned values to be ``NULL`` 
+in the result. This error occurs when you query a collection in which documents
+contain an array field that can be ``NULL`` and that array contains 
+an additional field that can also be ``NULL``. A workaround for this error is to 
+include the ``nonnullsched`` parameter with the ``createView`` method.
```

Comment:
Can we revise this to add context on how this workaround actually works?
Perhaps:

To resolve this error, create a view that contains only documents where the array field is an array
by including the ``nonnullsched`` parameter with the ``createView`` method.